registries/_format/sf-integer.md

```diff
@@ -0,0 +1,37 @@
+---
+owner: mikekistler
+issue: 
+description: structured fields integer as defined in [RFC 8941]
+base_type: integer
```

Comment:
Should be number

Reply:
Why base type for `sf-integer` should be `number`? I know it is restricted to integers that fit in a double, but semantically it is an integer. If "format" is just "extra information", then shouldn't we make the base type as specific as possible?

Reply:
@mikekistler `integer` is not a type in the JSON data model, it is just a convenience option in  JSON Schema's `type` keyword.  Formats are tied to a data model type, not the `type` keyword (this is a common misunderstanding).

Reply:
I've added `number` but kept `integer` because it is an allowed type in the OAS vocabulary.